### PR TITLE
Added my personal keymaps for dz60 and TMO50.

### PR DESCRIPTION
**Pull request by ottodokto**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [X] I have updated the documentation accordingly.
- [X] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [X] I have added tests to cover my changes.
- [X] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/dz60/keymaps/ottodokto/rules.mk [hunk 1]
```diff
@@ -0,0 +1,59 @@
+# MCU name
```
Comment (noroadsleft) [inline keyboards/dz60/keymaps/ottodokto/rules.mk]: The entire contents of this file can be minimized down to:

```make
VELOCIKEY_ENABLE = yes
```

The settings here override the settings in `keyboards/dz60/rules.mk`, but the Velocikey rule is the only thing you actually need; everything else will be inherited from keyboard level.

File: keyboards/tmo50/keymaps/ottodokto/keymap.c [hunk 2]
```diff
@@ -0,0 +1,57 @@
+#include QMK_KEYBOARD_H
+
+
+/*---default defines-------------------------------------------- */
+#define LT1_SPC     LT(1, KC_SPC)
+#define LT2_SPC     LT(2, KC_SPC)
+
+/*---RGB settings-------------------------------------------- */
```
Comment (noroadsleft) [inline keyboards/tmo50/keymaps/ottodokto/keymap.c]: Lines 8-17 should be in `keyboards/tmo50/keymaps/ottodokto/config.h`.

File: keyboards/tmo50/keymaps/ottodokto/rules.mk [hunk 3]
```diff
@@ -0,0 +1,63 @@
+# MCU name
```
Comment (noroadsleft) [inline keyboards/tmo50/keymaps/ottodokto/rules.mk]: Again, the only line you need in this file is:

```make
VELOCIKEY_ENABLE = yes 		# Enable velocikey.
```

**Comment by ottodokto** [comment]
HI, I've done a rebase of the branch and separated up the files per your suggestions. For some reason it seems that I accidentally deleted lufa, so I've resolved that. Thanks for your suggestions!

**Comment by ottodokto** [comment]
It looks like I actually messed up the directories for my keymaps, so I've updated it once again.

**Comment by ottodokto** [comment]
I accidentally messed up the directories for my keymaps, so I've fixed that accordingly.

File: keyboards/tmo50/keymaps/ottodokto/keymap.c [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+		KC_BRIU, KC_GESC,  KC_1, KC_2, KC_3, KC_4, KC_5, KC_6, KC_7, KC_8,    KC_9,   KC_0,    KC_MINS, KC_EQL, KC_DEL,
+		KC_BRID, KC_CAPS, _______, _______, _______, _______, _______, _______, KC_4, KC_5,    KC_6,   _______, _______, _______,
+		RGB_MOD, _______, _______, _______, _______, _______, KC_0, KC_1, KC_2, KC_3, _______, _______, _______, _______,
+		RGB_RMOD, _______, _______, _______, _______, _______, _______
+  ),
+  // FN & Navigation layer
+  [2] = LAYOUT_all(
+		KC_PGUP, _______,  KC_END, KC_UP, KC_HOME, _______, _______, _______, KC_F1, KC_F2,    KC_F3,   KC_F4,    _______, _______, _______,
+		KC_PGDN, _______, KC_LEFT, KC_DOWN, KC_RIGHT, _______, _______, _______, KC_F5, KC_F6,    KC_F7,   KC_F8, _______, _______,
+		KC_PSCREEN, _______, _______, _______, _______, _______, _______, KC_F9, KC_F10, KC_F11, KC_F12, _______, _______, _______,
```
Comment (noroadsleft) [inline keyboards/tmo50/keymaps/ottodokto/keymap.c]: Not a requirement or anything, but it seems like you went to some effort to use short-form (seven characters or fewer) keycodes in your keymaps, then you have this one random `KC_PSCREEN` one here.

```suggestion
		KC_PSCR, _______, _______, _______, _______, _______, _______, KC_F9, KC_F10, KC_F11, KC_F12, _______, _______, _______,
```

**Comment by ottodokto** [comment]
Good catch on the `KP_PRSC`. Thanks!

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**